i_scene_cp77_gltf/__init__.py

```diff
@@ -44,6 +44,7 @@ def execute(self, context):
 class CP77Import(bpy.types.Operator,ImportHelper):
     bl_idname = "io_scene_gltf.cp77"
     bl_label = "Import glTF"
+    bl_description = "Import Wolvenkit .glb/.gltf files" #Kwek: tooltips towards a more polished UI.
```

Comment:
Probably should say something like "Load glTF 2.0 files with Cyberpunk 2077 materials." or something similar. "Load a [type] file." is how Blender does it's native ones, for consistency. Also with a period at the end.

Reply:
Agreed. Will make the change.

Reply:
Also FYI--it needed no period, because Blender 3.1 adds it automatically.